lxd/daemon.go

```diff
@@ -1100,6 +1117,88 @@ func (d *Daemon) PasswordCheck(password string) error {
 	return nil
 }
 
+func (d *Daemon) ExpireLogs() error {
+	entries, err := ioutil.ReadDir(shared.LogPath())
+	if err != nil {
+		return err
+	}
+
+	result, err := dbContainersList(d.db, cTypeRegular)
+	if err != nil {
+		return err
+	}
+
+	newestFile := func(path string, dir os.FileInfo) time.Time {
+		newest := dir.ModTime()
+
+		entries, err := ioutil.ReadDir(path)
+		if err != nil {
+			return newest
+		}
+
+		for _, entry := range entries {
+			if entry.ModTime().After(newest) {
+				newest = entry.ModTime()
+			}
+		}
+
+		return newest
+	}
+
+	for _, entry := range entries {
+		// Check if the container still exists
+		if shared.StringInSlice(entry.Name(), result) {
+			// Remove any log file which wasn't modified in the past 48 hours
```

Comment:
I'm looking for a change that we can send to the stable branch, so adding new configuration knobs or adding new commands isn't an option here.

I could go with bumping that particular one to be 7 days, putting it in line with the standard logrotate policy.

But that still won't do you much good as lxc.log itself is lost after two container starts (lxc.log is moved to lxc.log.old on startup, so at the second start, the old lxc.log.old is lost).